server/data/ReportedAdjudicationResult.ts

```diff
@@ -22,5 +22,4 @@ export interface ReportedAdjudicationEnhanced extends ReportedAdjudication {
   formattedDateTimeOfIncident: string
   dateTimeOfIncident: string
   reportingOfficer?: string
-  prisonerReportUrl?: string
```

Comment:
removed this like we spoke about @richardpopple

Reply:
This now a parameter passed in from the controller?